src/types/UserscriptNameFormatter.d.ts

```diff
@@ -0,0 +1,3 @@
+import { UserscriptMetadata } from "./UserscriptMetadata"
```

Comment:
```suggestion
import type { UserscriptMetadata } from './UserscriptMetadata';
```
Minor nitpick for consistency...
I know that VS Code defaultas to double quotes unless the file already contains imports with single quotes (or you've changed the default setting).

P.S. Might be worth renaming this file to `BuildOptions.d.ts`, I could imagine having additional types for the other build functions (userscripts and bookmarklets) soon / in the future.

Reply:
Fixed and renamed to `BuildOptions.d.ts`.